eng/pipelines/templates/steps/publish-coverage.yml

```diff
@@ -9,10 +9,22 @@ steps:
     continueOnError: true
     condition: and(ne(variables['codecov-python-repository-token'], ''), succeededOrFailed(), ${{ parameters.RunCoverage }})
 
-  - task: PublishCodeCoverageResults@1
-    displayName: 'Publish Code Coverage to DevOps'
-    continueOnError: true
+  - pwsh: |
+      mv coverage.xml coverage-$(BuildTargetingString).xml; ls
```

Comment:
Why do you need to rename the file? 

If you do need to rename and you continue to use pwsh to do it please not use alias's like mv and ls as those might do things differently on different OS's. It is generally advised to use the ps cmdlets instead (see http://powershell-guru.com/powershell-best-practice-1-use-full-cmdlet-name-not-alias/).

Reply:
Some pipelines test multiple packages at once and these overwrite the coverage.xml file. I will change to the ps cmdlets, thanks for the suggestion.

Reply:
Is it possible to have the tool that produces the coverage report to output the file directly with the new name instead of needing to do a rename? If not then this should be fine.